CefGlue.Avalonia/Platform/Linux/XWindow.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Avalonia.Controls;
+using Avalonia.Platform;
+using System;
+using System.Runtime.InteropServices;
+
+namespace Xilium.CefGlue.Avalonia.Platform.Linux
+{
+    internal class XWindow : IHandleHolder, IPlatformHandle
+    {
+        [DllImport("libX11.so.6")]
+        public static extern int XDestroyWindow(IntPtr display, IntPtr w);
+
+        [DllImport("libX11.so.6")]
+        public static extern IntPtr XOpenDisplay([MarshalAs(UnmanagedType.LPUTF8Str)] string display_name);
+
+        [DllImport("libX11.so.6")]
+        public static extern IntPtr XCreateSimpleWindow(IntPtr display, IntPtr parent, int x, int y, uint width, uint height, uint border_width, UIntPtr border, UIntPtr background);
+
+        [DllImport("libX11.so.6")]
+        public static extern IntPtr XDefaultRootWindow(IntPtr display);
+
+        [DllImport("libX11.so.6")]
+        public static extern int XFlush(IntPtr display);
+
+        public IntPtr Handle { get; private set; }
+
+        public string HandleDescriptor => "XWindow";
+
+        private static IntPtr _display;
+        private readonly bool _needDispose;
+
+        public XWindow(IntPtr handle, bool needDispose)
+        {
+            Handle = handle;
+            _needDispose = needDispose;
+        }
```

Comment:
do not expose the need dispose param, otherwise it could be misused
```suggestion
        public XWindow(IntPtr handle) : this(handle, false) { }
        
        private XWindow(IntPtr handle, bool needDispose)
        {
            Handle = handle;
            _needDispose = needDispose;
        }
```